docs/core/tutorials/consuming-library-with-visual-studio.md

```diff
@@ -89,11 +88,13 @@ Just as you included unit tests in the same solution as your class library, you
 
 You can debug and publish the application that uses this library by following the steps in [Debugging your Hello World application with Visual Studio 2017](debugging-with-visual-studio.md) and [Publishing your Hello World Application with Visual Studio 2017](publishing-with-visual-studio.md).
 
-## Distributing the library in a NuGet package
+## Distribute the library in a NuGet package
 
 You can make your class library widely available by publishing it as a NuGet package. Visual Studio does not support the creation of NuGet packages. To create one, you use the [`dotnet` command line utility](../tools/dotnet.md):
 
-1. Open a console window. For example in the **Ask me anything** text box in the Windows taskbar, enter `Command Prompt` (or `cmd` for short), and open a console window by either selecting the **Command Prompt** desktop app or pressing Enter if it's selected in the search results.
+1. Open a console window.
+
+  For example, enter **Command Prompt** in the search box on the Windows task bar. Select the **Command Prompt** desktop app or press **Enter** if it's already selected in the search results.
```

Comment:
The next list item shows as 1 as well. Markdown require more spaces (not sure if it's 3 or 4). I'd make them 4 spaces.

```suggestion
    For example, enter **Command Prompt** in the search box on the Windows task bar. Select the **Command Prompt** desktop app or press **Enter** if it's already selected in the search results.
```

Reply:
Thanks for noticing this. It's fixed now.